website/project/views/node.py

```diff
@@ -962,10 +962,15 @@ def node_child_tree(user, node_ids):
                 'is_confirmed': contributor.is_confirmed
             })
 
+            affiliated_institutions = []
```

Comment:
This variable shouldn't be in the `for` loop. Also, the for loop below can be refactored into a list comprehension.

```python
affiliated_institutions_serialized = [{'id': ....} for institution in node.affiliated_institutions]
```